include/circt/Dialect/Calyx/CalyxPrimitives.td

```diff
@@ -417,6 +417,52 @@ def MulFOpIEEE754 : ArithBinaryFloatingPointLibraryOp<"ieee754.mul", [
   }];
 }
 
+// This models the division and square root, distinguished by `sqrtOp`,
+// operation interface in Berkeley HardFloat.
```

Comment:
```suggestion
// This models the division and square root operation interface in Berkeley HardFloat.
```

I imagine `$left` is the input, but what is `$right$` if `sqrtOp` is true?

Reply:
It's ignored:

> If sqrtOp is 0 when a new operation is started, the operation is division (a ÷ b). If sqrtOp is 1, the operation is the square root of a, and operand b is ignored.

Maybe I should add doc for it?